src/plugins/opensearch_dashboards_react/public/table_list_view/table_list_view.tsx

```diff
@@ -62,8 +64,11 @@ interface Item {
 
 export interface TableListViewProps {
   createItem?(): void;
+  dashboardItemCreators: () => DashboardCreators;
+  dashboardItemCreatorClickHandler: DashboardItemCreatorClickHandler;
```

Comment:
This doesn't seem right - the `TableListView` component is used for listing in both the `vizualize` app and the `dashboard` app. Adding dashboard specific required props in the common component breaks that generalization, and because they're required, I imagine it causes typescript errors for the usage in `visualize`. Why can't we leverage the existing `findItems` prop to collect all registered dashboard-like items upstream of this rendering component?

Reply:
Yes, you're right.  Adjusted code inside TableListView to accomodate null inputs.

The correct change would be to lift the Create button out of TableListView.  It was outside the scope of my change to accomplish that in the time I had to make this change.

Reply:
Create button has been extracted from TableListView.  It is now a prop/child component not required.  Currently only used by DashboardsList.